rasa/model.py

```diff
@@ -150,6 +150,10 @@ def get_model(model_path: Text = DEFAULT_MODELS_PATH) -> TempDirectoryPath:
     elif not model_path.endswith(".tar.gz"):
         raise ModelNotFound(f"Path '{model_path}' does not point to a Rasa model file.")
 
+    model_relative_path = create_output_path(model_path)
+    model_relative_path = os.path.relpath(model_relative_path)
+    logger.info(f"Loading model {model_relative_path}...")
+
```

Comment:
The Windows build is breaking because of https://bugs.python.org/issue7195. In some cases, the relative path on Windows may not exist. I suggest the following change:
```suggestion
    try:
        model_relative_path = os.path.relpath(model_path)
    except ValueError:
        model_relative_path = model_path
    logger.info(f"Loading model {model_relative_path}...")

```

I've also removed the call to `create_output_path` since it will return the same path it is given, when it ends with `.tar.gz`.

Reply:
Oh! We didn't know about that, thanks for warning us!

Reply:
Done! @federicotdn